testing/simapp/app.go

```diff
@@ -934,7 +934,7 @@ func (app *SimApp) GetKey(storeKey string) *storetypes.KVStoreKey {
 
 // GetStoreKeys returns all the stored store keys.
 func (app *SimApp) GetStoreKeys() []storetypes.StoreKey {
-	keys := make([]storetypes.StoreKey, len(app.keys))
+	keys := make([]storetypes.StoreKey, 0, len(app.keys))
```

Comment:
@coderabbitai I suspect there might be other occurrences in the codebase where slices are initialised with a non-zero length and then we append items to it. Can you check please?

Reply:
If both @DimitrisJim with his grep-fu skills and @coderabbitait with its AI powers agree on the answer, then I think we are good. :)

Reply:
uh, wait a minute, @DimitrisJim and @coderabbitai: check out [this commit](https://github.com/cosmos/ibc-go/pull/6444/commits/ae3f860db663f63701599bb382bce6185101286e), isn't these a couple of places where we should set zero length also?